test/test_hpe_icsp_os_deployment.py

```diff
@@ -200,13 +263,15 @@ def test_should_update_server_when_task_include_network_personalization(self):
         self.mock_icsp.common.monitor_execution.assert_has_calls(calls)
 
     def test_should_update_server_when_task_include_custom_attributes(self):
+        task_os_deployment = dict(TASK_OS_DEPLOYMENT, server_ipAddress=None)
```

Comment:
I'm seeing this code repeated all over the examples.

It wouldn't be better to add the server_ipAdress directly into the TASK_OS_DEPLOYMENT var?